projects/m2/010-random-lottery-numbers/js/index.js

```diff
@@ -0,0 +1,13 @@
+function ticket(){
+    const ticket = [];
+
+    while (ticket.length<6){
+        let number = Math.floor(Math.random()*(49-1)+1);
```

Comment:
**Don't use static code**, but where possible use **variables** and **constants**.
In some cases, even better if you pass them as parameters.
The best practice is always to have **functions** as **abstract** and **reusable** as possible.
For example:
`const numbers = 6;`
`maxValue = 49;`